### PR TITLE
Image refresh for rhel-8-7

### PR DESCRIPTION
**Pull request by cockpituous**
Image refresh for rhel-8-7
 * [x] image-refresh rhel-8-7

### CODE REVIEW

**Comment by martinpitt** [comment]
Loading VDO seems to kill the kernel. The "onboot timer" failure is known (this ran over night with a time zone day change confusion)

**Comment by martinpitt** [comment]
Reported https://bugzilla.redhat.com/show_bug.cgi?id=2117162

File: naughty/rhel-8/3724-vdo-kernel-hang-2 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Traceback (most recent call last):
```
Comment (marusak) [inline naughty/rhel-8/3724-vdo-kernel-hang-2:1]: This pattern is waay to generic. But I guess it is one of those tests that there is nothing else we can grasp on?

Comment (martinpitt) [reply naughty/rhel-8/3724-vdo-kernel-hang-2:1]: Unfortunately yes. It at least restricts it to the vdo test. See the [failing log](https://cockpit-logs.us-east-1.linodeobjects.com/pull-3722-20220809-231524-7e2e1e9d-rhel-8-7-distropkg-cockpit-project-cockpit/log.html#233-2). When it happens, the machine freezes completely.

I could include the `RuntimeError: Timed out on 'journalctl|gzip'` into the pattern, which would ensure "it's dead, Jim!"?

Comment (marusak) [reply naughty/rhel-8/3724-vdo-kernel-hang-2:1]: I don't think that would make it any more specific for this exact issue. So I am fine with this, thanks!

**Review by marusak — APPROVED** [review]

**Pull request merged**